include/signalrclient/signalr_value.h

```diff
@@ -12,7 +12,7 @@ namespace signalr
     /**
      * An enum defining the types a signalr::value may be.
      */
-    enum class type
+    enum class value_type
```

Comment:
Should we change the name of array to vector?

Reply:
It's not *necessarily* supposed to align with any particular language. We chose `map` over `object` because the concept of an "object" is really specific to JSON. Array vs vector, I could go either way.